src/main/java/org/jenkinsci/plugins/registry/notification/webhook/JSONWebHook.java

```diff
@@ -21,55 +21,33 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.jenkinsci.plugins.dockerhub.notification;
+package org.jenkinsci.plugins.registry.notification.webhook;
```

Comment:
No docker in this namespace seems a bit odd given it looks like it is docker specific?